### DISCUSSION TITLE
New Functionality and Many Codecs

### DISCUSSION THREAD

**Discussion by pschatzmann** [pschatzmann/arduino-audio-tools#324] [Announcements]
This release  took me quite some time since one of the codecs was holding me up. But finally I have everything working now and there are a couple of really cool features:

- Many new Codecs
- Support for VS1053 Codec Module
- Improvement and Cleanup of Desktop Build
- Format Converting Streams
- Support for Faust DSP
- Simple API for FFT / Alternative FFT Implementations
- 3 Band Equilizer
- MultiOutput to multiple Sinks
- Mixing multiple Sources
- Communication via RTSP  and  ESPNow
- PSRAM support for ESP32

<hr /><em>This discussion was created from the release <a href='https://github.com/pschatzmann/arduino-audio-tools/releases/tag/v0.9.0'>New Functionality and Many Codecs</a>.</em>

**Comment by mkoenings** [comment]
Hi Phil, I don't know how you do this, bringing out so many features in such a short time,  but this is absolutely amazing! It seems that I can create a combination of all kinds of audio/stream processing objects by just plugging them together with these brilliant Stream connectors. Amazing! Wireless, using compression, everything seems to work together...
I will start using this system from now on, be aware of a million questions coming to you :-)
Thanks,
Michael
